### ISSUE TITLE
Question about tradeReduced.

### ISSUE THREAD

**Issue by janckerchen** [unageanu/jiji2#399]
I  am not familiar with oanda api  and excuse me for some quetion about tradeReduced  field in order result.

If I *add* position for same instrument not reduce,  does api stilll return tradeReduced field?  or in tradeOpened? 

In test spec,  I found that a new position for same instrument is just added to internal array, rather than merge to one position.  so I dont know what will happen when I  add position for same instrument?

I checked  code  as  below,  but no anwser for me.

```ruby
 def apply_order_result(result, tick, agent = nil) # :nodoc:
      add(result.trade_opened, tick, agent) if result.trade_opened
      split(result.trade_reduced, agent) if result.trade_reduced
      result.trades_closed.each do |closed|
        close(closed.internal_id, closed.price, closed.timestamp, nil)
      end
      @account.update(self, tick.timestamp)
    end
```

**Comment by unageanu** [comment]
If you create a new position opposite to the one you currently have, the position will be closed or reduced.

- If you have 1000 USD/EUR sell position, and you buy 1500 USD/EUR, then 1000 USD/EUR sell position will be closed and 500 USD/EUR buy position will be created.
- If you have 1000 USD/EUR sell position, and you buy 500 USD/EUR, then 500 USD/EUR sell position will be reduced.

At that time, closed or reduced position information will be returned in `tradeReduced` or `tradeColosed` .

**Comment by janckerchen** [comment]
- If you have 1000 USD/EUR **buy** position, and you **buy** 500 USD/EUR more, what's the result returned?

**Comment by unageanu** [comment]
In this case, new 500 USD/EUR buy position will be created, and return position information in trade_opened, then tradeReduced will be nil, tradeColosed will be empty array.

**Comment by janckerchen** [comment]
I seemed to understand a little bit, for example:

- I have no position initially.
- then I buy 100 USD/EUR, return tradeOpened with internal_id_1
- then I buy 100 USD/EUR agian, return tradeOpened with internal_id_2 (new position as you said)
- then I buy 100 USD/EUR agian, return tradeOpened with internal_id_3
- I got three position by now.
- then I sell 250 USD/EUR
- so final action will return  two tradeColosed in array and one tradeReduced with 50

oanda dont merge position,  it's strange to me.
